acra-censor/handlers/blacklist_handler.go

```diff
@@ -28,14 +28,15 @@ func (handler *BlacklistHandler) CheckQuery(query string) (bool, error) {
 	if len(handler.queries) != 0 {
 		//Check that query is not in blacklist
 		if handler.queries[query] {
+			log.WithError(ErrQueryInBlacklist)
```

Comment:
this line doesn't make sense to me, I think this log will log nothing.

should be changed to
`log.WithError(ErrQueryInBlacklist).Errorln("Got error during parsing query")`